env.docker

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://daedalus-web-app-user:changeme@daedalus-web-app-db:5432/daedalus-web-app
+NUXT_R_API_BASE=http://daedalus-api:8001/
```

Comment:
> What's the new intended development workflow?
Sorry, not sure what you mean by this. These are different values from `.env.example` because they're referencing the container names rather than localhost, so we need to keep them separately from the ones the app uses when running outside docker.

As regards the ci script - tbh I can't actually remember why we're keeping NUXT_R_API_BASE in a github variable rather than just using the env we use in local dev. Was it something to do with `localhost` not working in CI but `127.0.0.1` does?

Reply:
I hadn't clocked that these values were different from env.example! So I'm no longer concerned about duplicating things.

I think my reason for using github variables was that you can change them without needing a whole PR, and so that I could have the CI variable not required to be the same as the local variable. But I'm not sure how you'd make github variables vary per environment (so that dev and staging and prod could have different values), so probably it's best if we move to using a set of files like `.env.local`, `.env.staging`, `.env.production` or something.

Reply:
You showed me the daedalus deploy [tool](https://github.com/jameel-institute/daedalus-deploy/pull/2) yesterday, which obviates the need for github action variables (it retrieves secrets from a vault - I forgot that one of the reasons I used github action variables was to avoid committing secrets to a public repo).

How shall we feed the NUXT_R_API_BASE environment variable to the Nuxt app for the various testing github action workflows? Currently there is a script `scripts/ci/copy-env-vars-to-dot-env-file` that copies out of github action variables into a new `.env` file. We could instead have a line like `mv .env.test .env` in the github action workflows.

Reply:
The e2e tests use env.docker. The integration tests could probably just re-use env.example as it's targeting localhost, and avoid needing the github variable..?

Reply:
It works. Can we merge that PR into this one now?